# This patch is here just so we don't have to make a p5-test-most port when
# all that's needed is Test::More.  <l2g@macports.org>
===============================================================================
--- t/override.t.orig	2011-03-24 11:11:19.000000000 -0700
+++ t/override.t	2011-03-24 11:11:40.000000000 -0700
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-use Test::Most tests => 4;
+use Test::More tests => 4;
 use lib 'lib';
 
 {
===============================================================================
--- META.yml.orig	2011-03-24 11:21:12.000000000 -0700
+++ META.yml	2011-03-24 11:21:31.000000000 -0700
@@ -3,7 +3,7 @@
 author:
   - "Curtis \"Ovid\" Poe <ovid@cpan.org>"
 build_requires:
-  Test::Most: 0.21
+  Test::More: 0.96
 configure_requires:
   Module::Build: 0.36
 generated_by: 'Module::Build version 0.3607'
===============================================================================
--- Build.PL.orig	2011-03-24 11:31:13.000000000 -0700
+++ Build.PL	2011-03-24 11:31:34.000000000 -0700
@@ -7,7 +7,7 @@
     license           => 'perl',
     dist_author       => 'Curtis "Ovid" Poe <ovid@cpan.org>',
     dist_version_from => 'lib/MooseX/Role/Strict.pm',
-    build_requires    => { 'Test::Most' => 0.21, },
+    build_requires    => { 'Test::More' => 0.96, },
     requires          => {
         'Moose' => 0.89,    # for --metaclass, -excludes
     },
